hack/feature-deploy.sh

```diff
@@ -18,6 +18,15 @@ export OC_TOOL="${OC_TOOL:-oc}"
 # expect kustomize to be in PATH by default
 KUSTOMIZE="${KUSTOMIZE:-kustomize}"
 
+echo "[INFO]: Pausing"
+# TODO patching to prevent https://bugzilla.redhat.com/show_bug.cgi?id=1792749 from happening
+# remove this once the bug is fixed
+mcps=$(${OC_TOOL} get mcp --no-headers -o custom-columns=":metadata.name")
```

Comment:
how would you feel about a hack/common.sh file that contains functions to pause/unpause. Then in those functions putting some resilience with regards to api failures? 

This isn't super critical, but if it saves us from hitting `/retest` a few times in the future, it would be worth it.

Reply:
Why not adding resilience here? 
It's the only place where we are pausing / unpausing and when the bug is fixed we are even removing it.

Reply:
yeah, i wouldn't block the pr on it or anything. just a thought if we encounter transient api failures related to this at some point.

Reply:
Added locally as this is throw away code.